torchvision/models/quantization/inception.py

```diff
@@ -0,0 +1,254 @@
+import sys
+import warnings
+from collections import namedtuple
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.models.inception
+from torch import Tensor
+from torch.jit.annotations import Optional
+from torchvision.models.utils import load_state_dict_from_url
+
+
+inception_module = sys.modules["torchvision.models.inception"]
+
+__all__ = [
+    "QuantizableInception3",
+    "inception_v3",
+    "QuantizableInceptionOutputs",
+    "_QuantizableInceptionOutputs",
+]
+
+
+model_urls = {
+    # Inception v3 ported from TensorFlow
+    "inception_v3_google": "https://download.pytorch.org/models/inception_v3_google-1a9a5a14.pth"
+}
+
+QuantizableInceptionOutputs = namedtuple(
+    "QuantizableInceptionOutputs", ["logits", "aux_logits"]
+)
+QuantizableInceptionOutputs.__annotations__ = {
+    "logits": torch.Tensor,
+    "aux_logits": Optional[torch.Tensor],
+}
+
+# Script annotations failed with _GoogleNetOutputs = namedtuple ...
+# _InceptionOutputs set here for backwards compat
+_QuantizableInceptionOutputs = QuantizableInceptionOutputs
+
+
+def inception_v3(pretrained_float_model=False, progress=True, **kwargs):
+    r"""Inception v3 model architecture from
+    `"Rethinking the Inception Architecture for Computer Vision" <http://arxiv.org/abs/1512.00567>`_.
+
+    .. note::
+        **Important**: In contrast to the other models the inception_v3 expects tensors with a size of
+        N x 3 x 299 x 299, so ensure your images are sized accordingly.
+
+    Args:
+        pretrained_float_model (bool): If True, returns a model pre-trained on ImageNet
+        progress (bool): If True, displays a progress bar of the download to stderr
+        aux_logits (bool): If True, add an auxiliary branch that can improve training.
+            Default: *True*
+        transform_input (bool): If True, preprocesses the input according to the method with which it
+            was trained on ImageNet. Default: *False*
+    """
+    if pretrained_float_model:
```

Comment:
If pretrained_float model is false, the code does not return a randomly initialized model, please check.

Reply:
I think line 76 does that: return QuantizableInception3(**kwargs), is it correct? This part of logic is the same as the original torchvision model.